tools/ak2-core.sh

```diff
@@ -292,6 +292,12 @@ flash_boot() {
   if [ "$(strings /tmp/anykernel/boot.img | grep SEANDROIDENFORCE )" ]; then
     printf 'SEANDROIDENFORCE' >> boot-new.img;
   fi;
+  if [ "$(grep_prop ro.product.brand)" == "lge" ] || [ "$(grep_prop ro.product.brand)" == "LGE" ]; then 
+    case $(grep_prop ro.product.device) in
+      d800|d801|d802|d803|ls980|vs980|101f|d850|d852|d855|ls990|vs985|f400) echo -n -e "\x41\xa9\xe4\x67\x74\x4d\x1d\x1b\xa4\x29\xf2\xec\xea\x65\x52\x79" >> boot-new.img;;
+    *) ;;
```

Comment:
Fairly certain this line can just go away since it's implied with case statements.

Reply:
Bump is already handled by custom_boot_image_patch.sh, just as it is by SuperSU: https://github.com/osm0sis/AnyKernel2/blob/master/tools/ak2-core.sh#L286-L291

Reply:
The supersu method doesn't work which would explain why that one doesn't either

Reply:
Oh? https://forum.xda-developers.com/lg-g2/general/bump-supersu-t3273251

Reply:
I understand that, so the point still stands, custom_boot_image_patch.sh is the way that it has been handled for quite some time. Chainfire's had this in production since SuperSU 2.6 and it works. If a user has a problem with it then they must be a newbie and be missing /data/custom_boot_image_patch.sh and should flash the zip from that thread to add it.

I would like to do a better job of it in the future by detecting it properly and then patching the new image, but this is not that way; it will require bbe (basically sed for binary data), and a rework of a number of things.